Add release info to "ReactOS in 2020" (#86)
reactos/web-content: content/project-news/reactos-in-2020.md

{{< figure link="/img/project-news/reactos-in-2020/reactos-pc98.png" caption="ReactOS running inside a PC-98 emulator" >}}
{{< /gallery >}}

## Polishing old Releases
 

> Due to the mentioned turbulences and the current 0.4.14RC being delayed longer than actually planned, Joachim Henze used the time to further improve the quality of older ReactOS releases, focusing mainly on further reducing the regression count.<br>
> 
> The changelog of his work can be seen here for each release branch:<br>
> 0.4.14-RC-58-g3461c59 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.14) (unreleased yet, still in Release-Candidate-phase)<br>
> 0.4.13-release-1-g2ac9d98 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.13) (still almost unaltered, improvement-phase starts after 0.4.14 will have been released)<br>
> 0.4.12-release-24-g4572c75 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.12)<br>
> 0.4.11-release-40-gf32c82e [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.11)<br>
> 0.4.10-release-49-gde68701 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.10)<br>
> 0.4.9-release-56-g9be0765 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.9)<br>
> 0.4.8-release-63-g1e9d797 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.8)<br>
> 0.4.7-release-75-g5f0c98f [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.7)<br>

Reviewer: ```suggestion
Due to the mentioned turbulences and the current 0.4.14-RC being delayed longer than actually planned, Joachim Henze used the time to further improve the quality of older ReactOS releases, focusing mainly on further reducing the regression count.

The changelog of his work can be seen here for each release branch:

- 0.4.14-RC-58-g3461c59 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.14) (unreleased yet, still in Release-Candidate-phase)
- 0.4.13-release-1-g2ac9d98 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.13) (still almost unaltered, improvement-phase starts after 0.4.14 will have been released)
- 0.4.12-release-24-g4572c75 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.12)
- 0.4.11-release-40-gf32c82e [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.11)
- 0.4.10-release-49-gde68701 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.10)
- 0.4.9-release-56-g9be0765 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.9)
- 0.4.8-release-63-g1e9d797 [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.8)
- 0.4.7-release-75-g5f0c98f [Changelog](https://git.reactos.org/?p=reactos.git;a=shortlog;h=refs/heads/releases/0.4.7)
```

Author: i will accept this first as there is no proper preview, then will scream and shout afterwards in case I won't like it.

> In sum 318commits to date! So there is still a lot improvement for you to explore!

Reviewer: ```suggestion
In sum 318 commits to date! So there is still a lot improvement for you to explore!
```

> You can download the isos at https://sourceforge.net/projects/reactos/files/ReactOS/

Reviewer: ```suggestion
You can download the ISOs at https://sourceforge.net/projects/reactos/files/ReactOS/
```
